ykcapi/src/lib.rs

```diff
@@ -34,7 +34,7 @@ pub extern "C" fn yk_new_location() -> Location {
 }
 
 #[no_mangle]
-pub extern "C" fn yk_drop_location(loc: *mut Location) {
+pub extern "C" fn yk_drop_location(loc: Location) {
```

Comment:
Begs the question: what's the point of this function now? If locations are stack alloc'd then they are automatically dropped, in both C and Rust.

Also, didn't the address of the location have some meaning? Maybe not any more, but if so, copying the location around by value will make its identity change.

Reply:
I see, there is drop code.

Identity question still holds.

Reply:
> Also, didn't the address of the location have some meaning?

No it deliberately doesn't. Internally, the malloc'd block the `Location` points to is significant, but it's a deliberate choice that users can move `Location`s around freely.

Reply:
OK, that's fine.